packages/docs/src/docs/pattern-documenting.md

```diff
@@ -55,6 +55,22 @@ We'd name our documentation file:
 atoms/button/button~red.md
 ```
 
+## Documenting Subgroups
+
+
+To document pattern subgroups, you need to create a companion `.md` file for that subgroup. For example create `_patters/atoms/buttons/_buttons.md` for a pattern subgroup. In the `.md` file, the above concept can be applied. The doc-file resolving works the following `{patternsRoot}/{pattern-group folder name}/{pattern-sub-group folder name}/_{pattern-sub-group raw name without prefixes}.md`
+
+
+```markdown
+---
+order: 0
+---
+
```

Comment:
That part can be removed from this documentation since it is already part of [Re-ordering pattern groups and subgroups](https://patternlab.io/docs/reorganizing-patterns/#heading-re-ordering-pattern-groups-and-subgroups)

Reply:
I was wondering of whether to include at least the one (and only ?) attribute for subgroups in this example, but you're perfectly right – as this is even already part of that reordering aspect, I've removed it from here.